assets/node-exporter/service-monitor.yaml

```diff
@@ -27,4 +27,3 @@ spec:
   selector:
     matchLabels:
       app.kubernetes.io/name: node-exporter
-      app.kubernetes.io/version: v0.18.1
```

Comment:
I am curious, why are we removing this downstream?

Reply:
This was an upstream change, not a downstream one.

Reply:
Yes, that's why I am asking why we are removing it downstream. I am aware why it was removed upstream (labels are immutable).